frontend/src/components/report/CSVReportFormatter.ts

```diff
@@ -119,9 +124,11 @@ export class CSVReportFormatter{
     }
     return result
   }
+
   private standard_csv_format(){
-    console.log("AAA", this.values_metadata)
+
     this.csv_content = "data:text/csv;charset=utf-8,";
+
     let headers_metadata = this.append_metadata_headers(this.csv_content)
     console.log('HEADERS META', headers_metadata)
```

Comment:
Same, probably we should remove those statements. Sometimes it's destructing when we debug something else and then there million of logs that are unnecessary

Reply:
Done